presto-main/src/main/java/com/facebook/presto/server/HttpRequestSessionContext.java

```diff
@@ -225,6 +236,12 @@ public boolean supportClientTransaction()
         return sessionProperties;
     }
 
+    private Set<String> parseClientTags(HttpServletRequest servletRequest)
+    {
+        Splitter splitter = Splitter.on(',').trimResults().omitEmptyStrings();
```

Comment:
This disallows tags with commas. Is that ok? Do we want any other restrictions?

Reply:
I think it's acceptable to disallow commas in tags.